### ISSUE TITLE
Create a warning that Safari is unsupported

### ISSUE THREAD

**Issue by Keavon** [GraphiteEditor/Graphite#183]
Create a way to tell the user that their browser is unsupported. Provide justification to avoid pissing people off. The reason is Safari's lack of support for the [BigInt64Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt64Array) API, because Safari is basically today's Internet Explorer.

It would probably be nice to still let the user view the web page even if functionality is broken. Let's do something that hides the page until the user reads and acknowledges the message. Try to implement this entirely in `App.vue` to avoid polluting other parts of the frontend codebase.

**Comment by tillarnold** [comment]
Are you looking for a solution with a proper modal dialog (which for example contains a link to download a supported browser) or would something like this suffice? 
```ts
if (!("BigInt64Array" in window)) {
	alert("Unfortunately your browser does not support the BigInt64Array API. To be able to use Graphite please use a supported browser such as Mozilla Firefox, Google Chrome or Microsoft Edge.");
}
```

**Comment by Keavon** [comment]
It is ideally better to avoid using the `alert(...)` feature (which blocks the window process). It would be preferable if App.vue can conditionally load the MainWindow component, and otherwise show a decently styled information dialog with a "see the interface anyways" button to load it.

**Comment by tillarnold** [comment]
I'll try to implement this.

**Comment by Keavon** [comment]
Thank you for the feature, that was an awesomely fast turnaround and it's nice and polished. Would you like some suggestions for next things you can tackle? Is your background in web, or do you also know some Rust?

**Comment by tillarnold** [comment]
Yeah it would be great if you could recomend a next task for me to tackle :smile:.  I do know some Rust, so I feel I should be able to do a Rust or a Rust/Web task.

**Comment by Keavon** [comment]
@tillarnold Splendid! It's especially great to have someone versed in both parts of the stack because a formidable percentage of all our tasks require touching frontend and backend in some capacity. I've picked out a few relevant possibilities for you which are ready to grab off the shelf. They're ordered from lesser to greater complexity:

- Copy and paste (and duplicate) layers #189
- Support for closing document tabs #211
- Differentiate keyboard shortcuts between Mac and non-Mac platforms #206 (Vue part only, to begin with)
- Polish the editor by clarifying to users when UI elements aren't implemented yet #196
- Add folders in the Layer Tree panel of the frontend #149
- Rearranging layers in the stack #141

Feel free to ask more details about each issue you're drawn to. I want to comment on the issue you choose and give you additional details and implementation steps so you aren't left guessing. Some will require a bit of design too.
